README.md

```diff
@@ -1,57 +1,105 @@
 <p align="center">
-  <a href="https://www.clerk.com/?utm_source=github&utm_medium=starter_repos&utm_campaign=clerk_remix_v2" target="_blank" align="center">
```

Comment:
Should I keep the UTM even if we are changing the name? I used the clerk-docs UTM from the other QuickStarts for that link

Reply:
Change the UTM to `clerk_remix_quickstart`

Reply:
Done! Matches the NextJS repos convention for this now, thanks 😄